### PR TITLE
feat: add a URL output type for `func describe`

### LINKED ISSUE #387: Need a programmatic way to get deployed function's URL
Knowing that a deployed function is actually a knative service, I can get its deployed URL using either `kn` or `kubectl`, but I'd prefer to use the same tool I deployed it with: `func`.

I'd like to be able to do something along these lines:

```shell
func deploy
curl $(func describe -o url)
```
I got the `url` output format idea from `kn service describe ...`

### PR DESCRIPTION
**Pull request by jcrossley3**
Fixes #387

### CODE REVIEW

File: cmd/describe.go [hunk 1]
```diff
@@ -161,3 +161,10 @@ func (d description) XML(w io.Writer) error {
 func (d description) YAML(w io.Writer) error {
 	return yaml.NewEncoder(w).Encode(d)
 }
+
+func (d description) URL(w io.Writer) error {
+	if len(d.Routes) > 0 {
+		fmt.Fprint(w, d.Routes[0])
```
Comment (zroubalik) [inline cmd/describe.go:167]:
```suggestion
		fmt.Fprintf(w, "%s\n", d.Routes[0])
```

**Review by zroubalik — COMMENTED** [review]
Looking good! Just a minor nit in output.

File: cmd/describe.go (hunk 1 above)
Comment (jcrossley3) [reply cmd/describe.go:167]: I was thinking the linefeed might mess up `curl $(func describe -o url)` but it doesn't seem to. Thanks!

**Review by zroubalik — APPROVED** [review]
LGTM

**Review by lance — APPROVED** [review]
TY!

**Pull request merged**